src/Symfony/Component/Mime/Encoder/IdnAddressEncoder.php

```diff
@@ -11,25 +11,21 @@
 
 namespace Symfony\Component\Mime\Encoder;
 
-use Symfony\Component\Mime\Exception\AddressEncoderException;
```

Comment:
`AddressEncoderException` is now not used anywhere anymore. But we cannot remove it due to BC?